lib/filters/genericnode.js

```diff
@@ -52,7 +52,8 @@ define(['helper'], function (helper) {
     return {
       run: run,
       setRefresh: setRefresh,
-      render: render
+      render: render,
+      unique: value + name
```

Comment:
@genofire i got no name and value to compare them. If you have a better idea, you are welcome :)